next.config.js

```diff
@@ -14,6 +15,13 @@ const noRedirectBlacklistedPaths = ['_next', 'api']; // Paths that mustn't have
 const publicBasePaths = ['robots', 'static', 'favicon.ico']; // All items (folders, files) under /public directory should be added there, to avoid redirection when an asset isn't found
 const noRedirectBasePaths = [...supportedLocales, ...publicBasePaths, ...noRedirectBlacklistedPaths]; // Will disable url rewrite for those items (should contain all supported languages and all public base paths)
 const date = new Date();
+const commitInfo = gitCommitInfo();
+const GIT_COMMIT_SHA = process.env.GIT_COMMIT_SHA || (commitInfo && commitInfo.commit);
```

Comment:
None of those logs print anything, sometimes it's undefined, sometimes empty

![Uploading image.png…]()